sample/src/main/java/org/aerogear/graphqlandroid/activities/MainActivity.kt

```diff
@@ -105,7 +119,31 @@ class MainActivity : AppCompatActivity() {
             customAlert.show()
 
         }
+    }
+
+    fun OfflineMutationSender() {
 
+        Log.e(TAG, "OfflineMutationSender inside")
+
+        val arrayList = myModel.getOfflineList()
+        Log.e(TAG, "OfflineMutationSender  :  ${arrayList.size} ")
+
+        arrayList.forEach {
+
+            Log.e(TAG, "OfflineMutationSender 1 : $it ")
+            val client = Utils.getApolloClient(this)?.mutate(
+                it
+            )?.refetchQueries(apolloQueryWatcher?.operation()?.name())
```

Comment:
Do we know what queries we want to refetch? How this works?

Reply:
@wtrocki Yes, apolloQueryWatcher is of that type we want to do the refetch Operation for.
Here we want to perform AllTaskQuery again after the mutation, so.
<img width="586" alt="Screenshot 2019-06-26 at 2 13 43 PM" src="https://user-images.githubusercontent.com/33238323/60165339-c26de900-981c-11e9-88ef-f593e0a8c003.png">